src/pages/practice/styles/PracticePage.css

```
@@ -209,11 +209,39 @@
 
-.record-button.is-recording {
-  background: #ef4b4b;
-  box-shadow: 0 10px 24px rgba(239, 75, 75, 0.35);
+/* 버튼 영역 (항상 동일 위치) */
+.practice-page__record-controls {
+  position: fixed;
+  right: 40px;
+  bottom: 40px;
+  z-index: 100;
+
+  display: flex;
```

Stylelint (17.6.0)

[error] Unexpected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)